### ISSUE TITLE
Feature: Increase the Identifier Size Limit

### ISSUE THREAD

**Issue by pluttrell** [sdkman/sdkman-cli#778]
[✔] I have read the [CONTRIBUTING guidelines](CONTRIBUTING.md)

**Feature request**
Currently the the length of the Identifier is limited to 15 chars. This limits the version numbering scheme, size of the numbers in the versions and/or the suffix (by convention). I'd like to request it be increased to 30 chars.

Here's one example when 15 is too short:
```
$ sdk install java 11.0.1.23.12-test /Library/Java/JavaVirtualMachines/.....
Invalid version! 11.0.1.23.12-test with length 17 exceeds max of 15!
```

**Comment by marc0der** [comment]
Hi @pluttrell, just some background about why this is restricted: it is in place so the list views don't breach their columns. I can see if we are able to push the limit somewhat for Java, as this has it's own list view these days.

**Comment by pluttrell** [comment]
Hi @marc0der, I thought that might be the case and was thinking we could just increase that column since it's at the far right.

**Comment by marc0der** [comment]
True, we can do this for the Java view, but SDKMAN always operates with in an 80 character width. We try to maintain this at all costs for consistency and great UX.

I remember we pushed the width for Java versions up to [17 characters](https://github.com/sdkman/sdkman-db-migrations/blob/master/src/main/scala/io/sdkman/changelogs/package.scala#L32), which isn't too much extra but does make a bit of a difference. I suspect that we forgot to update this portion of code to accommodate the extra characters in the local installation case.

If you felt comfortable dipping into some bash, a PR here would be greatly appreciated! The file where all this occurs is the [sdkman-install.sh](https://github.com/sdkman/sdkman-cli/blob/master/src/main/bash/sdkman-install.sh#L79).

**Comment by cstamas** [comment]
Same here
```
[cstamas@urnebes maven (master *)]$ sdk install maven 4.0.0-alpha-6-SNAPSHOT ~/tmp/apache-maven-4.0.0-alpha-6-SNAPSHOT
Invalid version! 4.0.0-alpha-6-SNAPSHOT with length 22 exceeds max of 15!
[cstamas@urnebes maven (master *)]$ 
```
